### ISSUE TITLE
List Back Links error (v3.0.2)

### ISSUE THREAD

**Issue by jonasferry** [SublimeText-Markdown/MarkdownEditing#628]
I can't list back links in my wiki pages. I don't remember if I've used it before, so I can't say how recent the issue is. Using the command palette, right-click-menu, or keyboard shortcut all give the below UnicodeDecodeError traceback, and nothing happens to the page.

Is this an issue with Sublime Text, the MarkdownEditing plugin, or with my setup? Please let me know if I need to provide more information on my system.

```
Traceback (most recent call last):
  File "C:\Users\ferry\Documents\sublime_text_build_4113_x64\Lib\python38\sublime_plugin.py", line 1518, in run_
    return self.run(edit)
  File "C:\Users\ferry\Documents\sublime_text_build_4113_x64\Data\Installed Packages\MarkdownEditing.sublime-package\plugins/wiki_page.py", line 23, in run
  File "C:\Users\ferry\Documents\sublime_text_build_4113_x64\Data\Installed Packages\MarkdownEditing.sublime-package\plugins/wiki_page.py", line 228, in find_files_with_ref
  File "C:\Users\ferry\Documents\sublime_text_build_4113_x64\Data\Installed Packages\MarkdownEditing.sublime-package\plugins/wiki_page.py", line 239, in contains_ref
  File "./python3.8/encodings/cp1252.py", line 23, in decode
UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 456: character maps to <undefined>
```

**Comment by deathaxe** [comment]
As related plugins have not been touched since last release, except being moved to another location, it might have  been an issue for a while. The currenct implementation uses default code page of the system (1252). I guess the related files are encoded using utf-8?

**Comment by jonasferry** [comment]
As troubleshooting, I moved all of my wiki page files to another folder and moved them back one by one and tested the command. After some trial and error I found that the "List back links" command worked for all pages except one. When I added that page to the folder, I got the above error when I ran the command on any page in the folder.

It turns out the page contained a curly double quote symbol that I probably copy-pasted from a PDF or something. I don't know if this will show up correctly on GitHub, but:

Offending symbol: ”
Updated symbol: "

When I manually changed to regular quotes, the "List back links" command works again for UTF-8 (my default page encoding), and also if I changed the encoding of the page to Windows 1252. I'd obviously prefer to not have had these issues, but now that I know the cause, I can avoid it.

This issue can be closed. I don't think that this plugin is the place to solve these encoding issues, unless the Python call to open a file can easily be made more encoding robust.
